scripts/qa_crowbarsetup.sh

```diff
@@ -1005,6 +1020,9 @@ EOF
         else
             complain 18 "You requested a HA setup but for this combination ($cloudsource : $slesdist) no HA setup is available."
         fi
+        if iscloudver 6plus; then
```

Comment:
As far as I understand, the `$slesdist` bit is only about the OS of the admin server, and this is kind of irrelevant, tbh. I'm not sure why it's there.